abi/type.go

```diff
@@ -186,8 +186,10 @@ type segment struct{ left, right int }
 // parseTupleContent splits an ABI encoded string for tuple type into multiple sub-strings.
 // Each sub-string represents a content type of the tuple type.
 // The argument str is the content between parentheses of tuple, i.e.
+//
 // (...... str ......)
-//  ^               ^
+//
+//	^               ^
```

Comment:
probably don't want this

Reply:
👍 I think it is because of go 1.19 formatter. I will leave a TODO here to revert the formatter change.